spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
"IMO, the Qualifier value should be mapped to the target of the bean injection"...I disagree there.  If that were really the case, we could just new the beans in the object itself...

In either case, that naming means that streams will only be made up of long running processes and tasks will only be made up of short running processes, an assumption I'm not 100% sold on.  It may be the case right now but I can definitely see the idea of running jobs as LRPs in certain use cases (cases where data is cached for example).  The idea that a module is just a boot jar should enable that flexibility.  I can also see the idea that certain pieces of a stream would be deployed as a task (although that may be a bit more complex if we do it at all).